### DISCUSSION TITLE
How to obtain the image of version 2.1.1

### DISCUSSION THREAD

**Discussion by A-little-bit-of-data** [apache/streampark#2848] [Q&A]
The 2.1.0 version was found in the docker warehouse, but the 2.1.1 version image was not found. Using the v2.1.1 version image, the pod keeps restarting.
![image](https://github.com/apache/incubator-streampark/assets/31812278/aa2301b2-4193-4324-8a51-0a36f2710218)
![image](https://github.com/apache/incubator-streampark/assets/31812278/c5b32d84-f579-49c4-b255-9d7a59622511)

**Comment by czy006** [comment]
Are there any error logs or other things？
